src/langcoder.hh

```diff
@@ -39,8 +39,9 @@ public:
 
   static quint32 findIdForLanguageCode3( std::string const & );
 
-  static QPair< quint32, quint32 > findIdsForName( QString const & );
-  static QPair< quint32, quint32 > findIdsForFilename( QString const & );
+  /// find id pairs like en-zh in dictioanry name
+  static QPair< quint32, quint32 > findLangIdPairFromStr( QString const & );
+  static QPair< quint32, quint32 > findLangIdPairFromPath( std::string const & );
```

Comment:
the parameter `std::string const &`  what about QString

Reply:
There is no need because that case doesn't exist :sweat_smile: